src/store/useProjectStore.ts

```diff
@@ -19,14 +19,24 @@ const defaultState: Omit<
 	| "projectList"
 	| "deleteProject"
 	| "createProjectFromManifest"
+	| "setActiveTab"
 > = {
 	projectsList: [],
+	activeTab: "",
```

Comment:
It's not exactly activeTab, because when we go to another page, it's not active. I think that `latestOpenedTab` or `lastSeenTab` might be better.

Reply:
changed on `latestOpenedTab`